src/worker/app.ts

```diff
@@ -2,21 +2,31 @@
 //        instance, mainly to use the `auth()` method. Probot needs refactored
 //        so that method is more easily accessible.
 
-import { createProbot } from 'probot';
+import {Application, createProbot} from 'probot';
 import { findPrivateKey } from 'probot/lib/private-key';
+import setupDeepcheck from "../deepcheck";
 
-const probot = createProbot({
+export const probot = createProbot({
   id: Number(process.env.APP_ID),
   cert: findPrivateKey(),
 
   // These aren't needed by worker process
   secret: undefined,
-  port: undefined,
+  port: Number(process.env.WORKER_PORT) || 8080,
   webhookPath: undefined,
   webhookProxy: undefined,
 });
 
+const App = async (app: Application): Promise<Application> => {
+  // Deepcheck and Healthchecks endpoints for checking app health
+  setupDeepcheck(app);
+  return app;
+};
+
```

Comment:
I had to add health-check endpoints to worker node, because it is required by Micros